specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -0,0 +1,16019 @@
+        "operationId": "VirtualMachineScaleSets_ConvertToSinglePlacementGroup",
+        "description": "Converts SinglePlacementGroup property to false for a existing virtual machine scale set.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine scale set to create or update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VMScaleSetConvertToSinglePlacementGroupInput"
+            },
+            "description": "The input object for ConvertToSinglePlacementGroup API."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "x-ms-examples": {
+          "VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MaximumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MaximumSet_Gen.json"
+          },
+          "VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MinimumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/setOrchestrationServiceState": {
+      "post": {
+        "tags": [
+          "VirtualMachineScaleSets"
+        ],
+        "operationId": "VirtualMachineScaleSets_SetOrchestrationServiceState",
+        "description": "Changes ServiceState property for a given service",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine scale set to create or update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/OrchestrationServiceStateInput"
+            },
+            "description": "The input object for SetOrchestrationService
```

Comment:
this type of information should go in the common systemData contract that all resources SHOULD be supporting in any new API version.  When does CRP plan to implement systemData for their resources?  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/16539/files

Reply:
@pilor
we are looking into the best course of action for this. 
However, I do want to note that this is our first time seeing the documentation and the 'SystemData' contract. How was this communicated before? CRP is expressing that this should be enforced at the Service/RP level than at the documentation phase.

Reply:
I've reached out to armpm to inquire how they are pushing RPs to implement this these days.  Does not look like it has been added as an s360 requirement yet (https://armwiki.azurewebsites.net/fundamentals/overview.html)

Reply:
@pilor CRP has decided they will continue with the "timeCreated" properties being added here and will implement systemData in the future